site/docs/pages/token/types.mdx

```diff
@@ -93,7 +93,18 @@ export type TokenSearchReact = {
 
 ```ts
 export type TokenSelectorReact = {
-  token: Token; // Rendered token
-  onClick: () => void; // Component on click handler
+  children: ReactElement<{ setToken: (token: Token) => void; onToggle: () => void }>; // Should either be a dropdown or modal that handle token selection
```

Comment:
Should this type `{ setToken: (token: Token) => void; onToggle: () => void }` being express in a more clean way? Maybe his own type?